src/pages/dashboard/_UsCumulativeDeathsContainer.js

```diff
@@ -36,11 +36,11 @@ export default function UsAreaChartContainer() {
   return (
     <section>
       <div>
-        <h4>Total cumulative deaths by day in the US</h4>
+        <h4>Daily deaths in the US</h4>
         <div>
           <BarChart
             data={data}
-            fill="#585BC1"
+            fill="#4C5559"
```

Comment:
we should use a color imported from the global color file `src/scss/colors.scss`